src/Symfony/Component/Ldap/Adapter/ExtLdap/ModificationInterface.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Ldap\Adapter\ExtLdap;
+
+
+interface ModificationInterface
+{
+    /**
+     * @return array
```

Comment:
should be removed

Reply:
Do you mean the whole Interface or only the docblock?